include/seqan3/range/view/get.hpp

```diff
@@ -0,0 +1,100 @@
+ *
+ * ### View properties
+ *
+ * This view is a **deep view:** Given a range-of-tuples as input (as opposed to just a range), it will apply
+ * the transformation on the innermost tuple (instead of the outermost range).
+ *
+ * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                     |
+ * |---------------------------------|:-------------------------------------:|:--------------------------------------------------:|
+ * | std::ranges::InputRange         | *required*                            | *preserved*                                        |
+ * | std::ranges::ForwardRange       |                                       | *preserved*                                        |
+ * | std::ranges::BidirectionalRange |                                       | *preserved*                                        |
+ * | std::ranges::RandomAccessRange  |                                       | *preserved*                                        |
+ * | std::ranges::ContiguousRange    |                                       | *lost*                                             |
+ * |                                 |                                       |                                                    |
+ * | std::ranges::ViewableRange      | *required*                            | *preserved*                                        |
+ * | std::ranges::View               |                                       | *preserved*                                        |
+ * | std::ranges::SizedRange         |                                       | *preserved*                                        |
+ * | std::ranges::CommonRange        |                                       | *preserved*                                        |
+ * | std::ranges::OutputRange        |                                       | *preserved*                                        |
+ * | seqan3::const_iterable_concept  |                                       | *preserved*                                        |
+ * |                                 |                                       |                                                    |
+ * | seqan3::reference_t             | std::tuple                            |                                                    |
+ *
+ * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
+ *
+ * ### Example
+ *
+ * \snippet test/snippet/range/view/get.cpp usage
+ * \hideinitializer
+ */
+template <size_t index>
+inline auto const get = view::transform([] (auto && in)
+{
+    using std::get;
+    // if constexpr (std::is_lvalue_reference_v<decltype(in)>)
+        // return static_cast<std::tuple_element_t<remove_cvref_t<decltype(in)>, index> &>(get<index>(std::forward<decltype(in)>(in)));
+    // else
+    // TODO: Currently does not return a reference.
+    return get<index>(std::forward<decltype(in)>(in));
```

Comment:
add a static assertion here that enforces the tuple_like_concept
```cpp
    static_assert(tuple_like_concept<decltype(in)>,
                  "You may only pass ranges to view::get whose reference_t models the tuple_like_concept.");
    return std::get<index>(in);
```
You need to add the include for `core/concept/tuple.hpp`.

For reasons unclear to me, you need to also adapt `core/concept/tuple.hpp` to include `core/pod_tuple.hpp` (while you are there you can remove the duplicate include-lines)..